packages/react/src/components/dynamic-flow/configured-handle-form.tsx

```diff
@@ -15,11 +15,13 @@ import {
 export const TAB_NAME = {
   email: "email",
   phone: "phone",
+  username: "username",
```

Comment:
Does it make sense to stick to tabs here? Won't this be a little bit too cramped?

Reply:
I'd leave this open to feedback as usernames are opt in and we don't expect customers to run all three possible handle types at the same time.